CMakeLists.txt

```diff
@@ -244,7 +244,11 @@ endif ()
 
 # Disable optimizations for interop.cpp for all compilers, to allow optimized
 # builds without need for -fno-omit-frame-pointer
-set_source_files_properties("${CMAKE_CURRENT_SOURCE_DIR}/src/OpenLoco/Interop/Interop.cpp" PROPERTIES COMPILE_FLAGS "-fno-omit-frame-pointer -O0")
+if (MSVC)
+    set_source_files_properties("${CMAKE_CURRENT_SOURCE_DIR}/src/OpenLoco/Interop/Interop.cpp" PROPERTIES COMPILE_FLAGS "/Oy- /O0")
```

Comment:
pretty sure it should be `/Ody`.    `/O0` means nothing.
https://docs.microsoft.com/en-us/cpp/build/reference/o-options-optimize-code?view=msvc-160

Reply:
Fixed this